app/views/unauthorised.scala.html

```diff
@@ -14,12 +14,58 @@
  * limitations under the License.
  *@
 
-@this(mainTemplate: main_template)
+@import config.ExternalServicesConfig
+
+@this(mainTemplate: main_template, config: ExternalServicesConfig)
```

Comment:
Could you add a view spec for this page?

Probably possible copy paste from dec service

Reply:
There already was a view spec for the page.  Added tests for the three links on the page.
Also ported over the "AllMessageKeysAreMandatoryMessages" class to the base spec so that all view specs are now asserting that their message keys exist.